packages/gestalt/src/__snapshots__/ScrollableContainer.test.js.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`ScrollableContainer ScrollableContainer correctly sets height and overflow props 1`] = `
 <div
-  className="box overflowScroll relative"
+  className="box overflowScroll paddingXundefined paddingYundefined relative"
```

Comment:
This seems wrong?

Reply:
looking now...

Reply:
OK, added a default to 0. 

I was using this pattern 

`padding={padding}`. Without default it creates `paddingXundefined paddingYundefined` Now we will have paddingX0 paddingY0. I also added some tests for that.